_docs/technical/protocols/2017-01-13-binary.md

```diff
@@ -157,9 +159,11 @@ newtype TinyVarInt = TinyVarInt {getTinyVarInt :: Word16}
     deriving (Eq, Ord, Show, Generic, NFData)
 ~~~
 
+[Source code link](https://github.com/input-output-hk/cardano-sl/blob/895619d7056fe397e9c2a56d88994f167263c397/core/Pos/Binary/Class.hs#L295).
+
 | Field size | Type           | Description                            |
 |------------|----------------|----------------------------------------|
-|        1-2 | UVarInt Word16 | Variable length integer up to 2^14 - 1 |
+| 1-2        | UVarInt Word16 | Variable length integer up to 2^14 - 1 |
```

Comment:
If some types doesn't have message type limit then maybe put information about this? For example:

`Message size limit: unspecified (as a part of <other data type>)` or something like that